compiler/compile/VirtualGuard.hpp

```diff
@@ -64,7 +65,8 @@ enum TR_VirtualGuardTestType
    TR_VftTest,
    TR_MethodTest,
    TR_NonoverriddenTest,
-   TR_RubyInlineTest
+   TR_RubyInlineTest,
+   TR_FSDTest
```

Comment:
Would be good to document what this is somewhere (either above the enum or at the enum some how.